llvm/lib/Target/X86/X86InstrAsmAlias.td

```diff
@@ -124,19 +124,31 @@ def : InstAlias<"ccmp"#Cond#"{q} $dcf\t{$src2, $src1|$src1, $src2}",
 defm : CCMP_Aliases<"o" ,  0>;
 defm : CCMP_Aliases<"no",  1>;
 defm : CCMP_Aliases<"b" ,  2>;
+defm : CCMP_Aliases<"c" ,  2>;
```

Comment:
What does `c` mean?

Reply:
`carry`

Reply:
https://www.godbolt.org/z/G1nv15xar

Reply:
Why do you want to add it?

Reply:
First, binutil on godbolt is too old. binutil has updated to 2.43, while it only has 2.38

Reply:
My objective is to align with gas.

Reply:
binutils reference: https://github.com/bminor/binutils-gdb/blob/b935898fd1b323a47452e672de597a497bdeadf6/opcodes/i386-opc.tbl#L350